### ISSUE TITLE
Sparks accordions should use <button> as their triggering elements

### ISSUE THREAD

**Issue by RCopeland** [sparkdesignsystem/spark-design-system#2724]
Currently spark's accordions use <a> when, since they don't navigate, should be <buttons>. See rm-12 in the a11y audit.
